components/discover/LeftOverlay.tsx

```
@@ -0,0 +1,32 @@
+import { cn } from "@/lib/utils";
+
+const LeftOverlay = ({ username, tags }: { username: string, tags: string[] }) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <div className="absolute bottom-24 left-4 text-white max-w-[80%]">
+      <h3 className="font-bold text-lg">@{username}</h3>
+      <div
+        className="flex gap-2 items-center cursor-pointer"
+        onClick={() => setIsExpanded(!isExpanded)}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)